fab/operations/release.py

```diff
@@ -281,3 +281,16 @@ def mark_keep_until(keep_days):
     until_date = (datetime.utcnow() + timedelta(days=keep_days)).strftime(DATE_FMT)
     with cd(env.code_root):
         sudo('touch {}{}'.format(KEEP_UNTIL_PREFIX, until_date))
+
+
+@roles(ROLES_ALL_SRC)
+@parallel
+def apply_patch(filepath):
+    destination = '/home/{}/patch.patch'.format(env.user)
```

Comment:
yeah i was thinking that if we did this multiple times we might want a record on the servers of what had happened. and there would be less likelihood of bugs related to applying the wrong patch (not sure how that would happen but was trying to be paranoid)